CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 Anyone can contribute to RSWG; you don't have to join the Working Group, because there is no "membership" -- anyone who participates in the work, as outlined below, is part of the RSWG Working Group.
 
-Before doing so, it's a good idea to familiarize yourself with our [charter](https://datatracker.ietf.org/rfcedtyp/rswg/about/). If you're new to this, you may also want to read [an informal guide to the IETF process](https://www.ietf.org/standards/process/informal/) and the [Tao of the IETF](https://www.ietf.org/tao.html).
+Before doing so, it's a good idea to familiarize yourself with our [charter](https://datatracker.ietf.org/edwg/rswg/about/). If you're new to this, you may also want to read [an informal guide to the IETF process](https://www.ietf.org/standards/process/informal/) and the [Tao of the IETF](https://www.ietf.org/tao.html).
```

Comment:
```suggestion
Before doing so, it's a good idea to familiarize yourself with our [charter](https://datatracker.ietf.org/group/rswg/about/). If you're new to this, you may also want to read [an informal guide to the IETF process](https://www.ietf.org/standards/process/informal/) and the [Tao of the IETF](https://www.ietf.org/tao.html).
```

Reply:
Using the generic "group" selector instead of the group_type selector is far more future-proof.